vivisect/cli.py

```diff
@@ -215,7 +215,33 @@ def do_xrefs(self, line):
         -T Show xrefs *to* the given address
         -F Show xrefs *from* the given address (default)
         """
-        pass
+        parser = e_cli.VOptionParser()
+        parser.add_option('-T', action='store_true', dest='xrto')
+        parser.add_option('-F', action='store_true', dest='xrfrom')
+        argv = shlex.split(line)
+        try:
+            options, argv = parser.parse_args(argv)
+        except Exception as e:
+            self.vprint(repr(e))
+            return self.do_help('xrefs')
+
+        if len(argv) < 1:
+            self.vprint('Supply a va_expr')
+            return self.do_help('xrefs')
+
+        va = self.parseExpression(argv[0])
+
+        fptr = []
+        if options.xrto:
+            fptr.append(self.getXrefsTo)
+        if options.xrfrom:
+            fptr.append(self.getXrefsFrom)
+
+        for func in fptr:
+            for xrfr, xrto, rtype, rflags in func(va):
+                tname = ref_type_names.get(rtype, 'Unknown')
+                self.vprint('\tFrom: {}, To: {}, Type: {}, Flags: {}'. format(hex(xrfr), hex(xrto), tname, hex(rflags)))
```

Comment:
we have typically used old-school printf-style formatting:
    self.vprint('\tFrom: 0x%x, To: 0x%x, Type: %s, Flags: 0x%x' % (xrfr, xrto, tname, rflags))

is there reason to choose one over the other?  how should we handle the different mechanisms going forward?  do we just embrace whichever one makes sense at the time to the particular contributor?  or should we standardize on one and make exceptions for when the other is a better fit?